qiskit/pulse/instructions/instruction.py

```diff
@@ -61,6 +60,8 @@ def __init__(self,
 
         Raises:
             PulseError: If duration is negative.
+            PulseError: If the input ``channels`` are not all of
+                type :class:`Channel`.
```

Comment:
```suggestion
            PulseError: If the input ``channels`` are not all of type :class:`Channel`.
```

Reply:
My linter prefers what I have currently.

Reply:
Why doesn't it fit on one line?